openSpotServer/parkingSpot/parkingSpots.py

```diff
@@ -0,0 +1,7 @@
+import numpy as np
+
+# 					  [x,y]
+#                     Top Left, Top Right,  Bottom Right, Bottom Left
+spot_15 = np.array([[1465,152], [1700,152], [1700,248],   [1430,248]], np.int32)
```

Comment:
Just curious Why start with spot_15,16 ...

Reply:
It's the actual spot number. If you look closely at the picture the spots have numbers to them so I just labeled it accordingly.

Reply:
These are just temporary, will have to redo spots once camera mount is 3D printed.

Reply:
Cool